rsyncd/rsyncd.go

```diff
@@ -190,7 +202,7 @@ func (s *Server) HandleDaemonConn(ctx context.Context, conn io.ReadWriter, remot
 	log.Printf("client %v requested rsync module %q", remoteAddr, requestedModule)
 	module, err := s.getModule(requestedModule)
 	if err != nil {
-		fmt.Fprintf(cwr, "@ERROR: Unknown module '%s'\n", requestedModule)
+		fmt.Fprintf(cwr, "@ERROR: Unknown module %q\n", requestedModule)
```

Comment:
I also changed this for the `%q` variant. This affects the protocol (single quote -> double quote), so please review carefully.

Tests didn't seem to break.

Reply:
I think the protocol generally doesn’t mind what comes after the @ERROR:, but thanks for flagging this :)